index.html

```diff
@@ -80,7 +80,109 @@ <h1 class="mn__title">
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+        <div class="proj__container" id="proj__3">
+        <img class="proj__img" src="img/Work 3.webp" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+        <div class="proj__container" id="proj__4">
+        <img class="proj__img" src="img/Work 4.jpg" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+        <div class="proj__container" id="proj__5">
+        <img class="proj__img" src="img/Work 5.webp" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+        <div class="proj__container" id="proj__6">
+        <img class="proj__img" src="img/Work 6.jpg" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+      </section>
+      <section class="abt__me">
+        <div class="abt__me__title"><h2>About me</h2></div>
+         <div id="abt__txt">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, <br> don't hestiate to contact me.
+          </p>
+        </div>
+        <button class="see__proj__btn">Get My Resume</button>
```

Comment:
- [ ] I would like to recommend you to use `<a>` instead `<button>` here because it will allow you to link it to your own Resume in the future :+1: